### ISSUE TITLE
Simple question about validation while training

### ISSUE THREAD

**Issue by arcyfelix** [tflearn/tflearn#855]
Hi, I would like to ask how TFLearn picks up the validation set when it is given as a fraction of the training dataset, f.e. 0.1.
Is it a random choice or are the examples picked up in a way that all the classes are balanced inside the training and validation datasets?

**Comment by BBarbosa** [comment]
Hi @arcyfelix !
That question was already answered. Check here #187 
Hope it helps!
